cloudigrade/util/aws/cloudtrail.py

```diff
@@ -0,0 +1,130 @@
+"""Helper utility module to wrap up common AWS CloudTrail operations."""
+import logging
+
+from botocore.exceptions import ClientError
+from django.conf import settings
+from django.utils.translation import gettext as _
+
+
+logger = logging.getLogger(__name__)
+
+
+def configure_cloudtrail(session, aws_account_id,
+                         include_global_service_events=True,
+                         is_multi_region_trail=True):
+    """
+    Configure a CloudTrail in the customer account.
+
+    Args:
+        session (boto3.Session): A temporary session tied to a customer account
+        aws_account_id (string): The aws account id to include in the name
+        include_global_service_events (bool): Specifies whether trail is
+            publishing events from global services to log files
+        is_multi_region_trail (bool): Specifies whether the trail is created in
+            all regions or current region
+
+    Returns:
+        The response code of creating cloudtrail.
+
+    """
+    cloudtrail = session.client('cloudtrail')
+    name = f'cloudigrade-{aws_account_id}'
+
+    if lookup_trail(cloudtrail, name):
+        response = update_cloudtrail(cloudtrail, name,
+                                     include_global_service_events,
+                                     is_multi_region_trail)
+    else:
+        response = create_cloudtrail(cloudtrail, name,
+                                     include_global_service_events,
+                                     is_multi_region_trail)
+    # limit to write only events & turn on logging
+    put_event_selectors(cloudtrail, name)
+    cloudtrail.start_logging(Name=name)
+    return response
+
+
+def lookup_trail(cloudtrail, name):
```

Comment:
I would suggest a name like has_trail() since the function returns a boolean rather than an a trail object.

Reply:
This makes sense! I will change it.